hs_core/templates/pages/genericresource.html

```diff
@@ -1357,7 +1358,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 var t = $('#extraMetaTable').DataTable();
                 var row_to_edit = t.row("#" + row_id_str);
                 var oldname = row_to_edit.data()[0];
-                var oldvalue = row_to_edit.data()[1];
+                var oldvalue = $("<div/>").html(row_to_edit.data()[1]).text();
```

Comment:
@mauriel133 This is related to the comment below on adding ```.trim()``` call. I am not sure whether it is needed or not. But I think we need to keep the consistency meaning either we don't add ```.trim()``` call as you have done currently, or if you add ```.trim()``` call to the other place, make sure you do the same thing here as well to keep them consistent.